src/Migration/Migration.php

```diff
@@ -61,6 +62,11 @@ public static function migrateAll() {
 					'complete' => 0,
 					'failed'   => 0,
 				],
+				'restrictions' => [
+					'index'    => 0,
```

Comment:
Ui, da hast du die Migration geschrieben, aber in den Migrationen von CB1 -> CB2. CB1 hatte (vanilla) gar keine Restrictions. Deshalb müsste die Migration irgendwo anders erfolgen.